explainshell/config.py

```diff
@@ -8,7 +8,7 @@
 
 MAN2HTML = os.path.join(TOOLSDIR, 'w3mman2html.cgi')
 
-MONGO_URI = 'mongodb://localhost'
+MONGO_URI = os.getenv('MONGO_URI', 'mongodb://localhost')
```

Comment:
Instead of adding all these environment variables, I'm suggesting that a setup phase in docker (not sure what it's called) will write a new config.py that replaces the existing one, with the new values for the things it cares about, like MONGO_URI (and HOST it seems).

As a middle ground, I guess we could add something like this at the end of config.py:

```
# load additional config from an out-of-tree file
if os.getenv('ADDITIONAL_CONFIG'):
  execfile(os.getenv('ADDITIONAL_CONFIG'))
``` 

Then you can override the values you want in another config file and set the environment variable once.